lc3/include/sw_codec_lc3.h

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: Nordic-5-Clause
```

Comment:
Can we use the short 5-Clause license?
It looks like lc3/Cmakelists.txt uses the short one